ddtrace/llmobs/_llmobs.py

```diff
@@ -30,11 +30,14 @@
 from ddtrace.llmobs._constants import OUTPUT_DOCUMENTS
 from ddtrace.llmobs._constants import OUTPUT_MESSAGES
 from ddtrace.llmobs._constants import OUTPUT_VALUE
+from ddtrace.llmobs._constants import PARENT_ID_KEY
+from ddtrace.llmobs._constants import PROPAGATED_PARENT_ID_KEY
 from ddtrace.llmobs._constants import SESSION_ID
 from ddtrace.llmobs._constants import SPAN_KIND
 from ddtrace.llmobs._constants import SPAN_START_WHILE_DISABLED_WARNING
 from ddtrace.llmobs._constants import TAGS
 from ddtrace.llmobs._trace_processor import LLMObsTraceProcessor
+from ddtrace.llmobs._utils import _get_llmobs_parent_id
 from ddtrace.llmobs._utils import _get_ml_app
 from ddtrace.llmobs._utils import _get_session_id
 from ddtrace.llmobs._writer import LLMObsEvalMetricWriter
```

Comment:
I'm debating on if we should add `requests/httpx/urllib3/...` integrations to the list of supported integrations, since users will need to have these enabled in order for distributed tracing to work correctly. Thoughts @lievan ?

Reply:
Let's patch these in enable() but not list it as a supported integration?

Reply:
As per offline discussion - we will have the same functionality as `ddtrace-run` where we call `patch_all()` for non-LLMObs integrations. For users that use libraries that aren't patched by default by `patch_all()` (ex: urllib3), we will add documentation in our corp docs to explicitly call `ddtrace.patch(urllib3=True)`.